syntax = "proto3";

package oauth_proto;

service ResourceService {
    rpc GetResource(ResourceRequest) returns (ResourceResponse) {}
}

message ResourceResponse {
    string username = 1;
    string first_name = 2;
    string last_name = 3;
}

message ResourceRequest {
}
